routes/users.js

```diff
@@ -209,4 +210,28 @@ router.get('/isUsernameAvailable/:username', authenticate, usersController.getUs
  */
 router.get('/:username', usersController.getUser)
 
+/**
+ * @swagger
+ * /stocks/self:
+ *  get:
+ *   summary: Used to get all the stocks
+ *   tags:
+ *     - Users Stocks
+ *   responses:
+ *     200:
+ *       description: returns stocks of the user
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/userStocks'
+ *     500:
+ *       description: badImplementation
+ *       content:
+ *         application/json:
+ *           schema:
+ *             $ref: '#/components/schemas/errors/badImplementation'
+ */
+
+router.get('/stocks/self', authenticate, getUserStocks)
```

Comment:
This should ideally be in a separate routes file

Reply:
Ok Sure, will move it into `stocks` routes

Reply:
d0ce9ca